documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -403,16 +403,15 @@ For more information about setting up and deploying Prometheus and Grafana, see
 [id='con-common-configuration-jvm-{context}']
 === `jvmOptions`
 
-The following components of Strimzi run inside a Virtual Machine (VM):
+The following Strimzi components run inside a Virtual Machine (VM):
```

Comment:
I think we should mention Java to avoid any possible confusion with regular VMs.

```suggestion
The following Strimzi components run inside a Java Virtual Machine (JVM):
```